lib/main_containers.pm

```diff
@@ -321,6 +321,12 @@ sub load_helm_chart_tests {
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIMES');
 
+    # Workaround while we use 15.99 for SLE 16 maintenance
+    if (check_var('VERSION', '15.99')) {
+        set_var('VERSION', '16.0');
+        testapi::set_distribution(DistributionProvider->provide());
```

Comment:
I don't think this is required, that is from different commit.
But rest will do what you need.